2_Controllers/test_controller.js

```diff
@@ -60,11 +63,25 @@ const postComments = (request, response, next) => {
     .catch(next);
 };
 
+const patchArticleVotes = (request, response, next) => {
+  const { article_id } = request.params;
+  const { inc_votes } = request.body;
+  if (typeof inc_votes !== "number") {
```

Comment:
this is not necessary as when inc_votes is not a number when you try re-assign votes to votes + inc_votes they both need to be numbers so sql will throw an error